### PR TITLE
alertnow integration

### PR DESCRIPTION
**Pull request by Uijong-Kim**
### What does this PR do?

We proceed with the initial upload to add content for the AlertNow service to Datadog Integration.

### Motivation

To add AlertNow service to Datadog Integration.

### Review checklist

- [V] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [V] Feature or bugfix has tests
- [V] Git history is clean
- [V] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

No. thanks

### CODE REVIEW

**Comment by alai97** [comment]
Created DOCS-3458 for review!

**Comment by sarah-witt** [comment]
Hi @Uijong-Kim, thank you for the contribution. For your integration to display in the Datadog app, can you please create a manifest.json file: https://docs.datadoghq.com/developers/integrations/check_references/?tab=manifestversion2#manifest-file . You can create this automatically by recreating the integration with the `ddev create integration --type tile` command and then updating the newly created readme with your information.

**Comment by buraizu** [comment]
Hi @Uijong-Kim, just following up once again on this pull request. As mentioned by my colleague, for your integration to display in the Datadog app, you will need to create a manifest.json file: https://docs.datadoghq.com/developers/integrations/check_references/?tab=manifestversion2#manifest-file . 
You can create this automatically by recreating the integration with the ddev create integration --type tile command and then updating the newly created readme with your information.
Thanks, and let us know if you have any questions!

**Comment by Uijong-Kim** [comment]
Hi, I tried running the command below, but only an error message is being displayed. 
What should I do to fix the error or can you provide a link to the documentation I can refer to?

ddev create integration --type tile

Traceback (most recent call last):
  File "/opt/homebrew/bin/ddev", line 8, in <module>
    sys.exit(ddev())
  File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 1130, in __call__
    return self.main(*args, **kwargs)
  File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 1055, in main
    rv = self.invoke(ctx)
  File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 1657, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 1404, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 760, in invoke
    return __callback(*args, **kwargs)
  File "/opt/homebrew/lib/python3.9/site-packages/click/decorators.py", line 26, in new_func
    return f(get_current_context(), *args, **kwargs)
  File "/opt/homebrew/lib/python3.9/site-packages/datadog_checks/dev/tooling/commands/create.py", line 164, in create
    files = create_template_files(integration_type, root, config, read=not dry_run)
  File "/opt/homebrew/lib/python3.9/site-packages/datadog_checks/dev/tooling/create.py", line 154, in create_template_files
    file_path = f'{new_root}{file_path.format(**config)}'
UnboundLocalError: local variable 'file_path' referenced before assignment

**Comment by Uijong-Kim** [comment]
> Hi, I tried running the command below, but only an error message is being displayed. If the "--type tile" option is not added, files and directories are created normally, but if the option is added and executed, the following error message is output. What should I do to fix the error or can you provide a link to the documentation I can refer to?
> 
> ddev create integration --type tile
> 
> Traceback (most recent call last): File "/opt/homebrew/bin/ddev", line 8, in sys.exit(ddev()) File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 1130, in **call** return self.main(*args, **kwargs) File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 1055, in main rv = self.invoke(ctx) File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 1657, in invoke return _process_result(sub_ctx.command.invoke(sub_ctx)) File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 1404, in invoke return ctx.invoke(self.callback, **ctx.params) File "/opt/homebrew/lib/python3.9/site-packages/click/core.py", line 760, in invoke return __callback(*args, **kwargs) File "/opt/homebrew/lib/python3.9/site-packages/click/decorators.py", line 26, in new_func return f(get_current_context(), *args, **kwargs) File "/opt/homebrew/lib/python3.9/site-packages/datadog_checks/dev/tooling/commands/create.py", line 164, in create files = create_template_files(integration_type, root, config, read=not dry_run) File "/opt/homebrew/lib/python3.9/site-packages/datadog_checks/dev/tooling/create.py", line 154, in create_template_files file_path = f'{new_root}{file_path.format(**config)}' UnboundLocalError: local variable 'file_path' referenced before assignment

i solved this problem

**Review by Uijong-Kim — COMMENTED** [review]
I checked and added the necessary files.

**Comment by Uijong-Kim** [comment]
@sarah-witt @buraizu I have finished creating the manifest file.

File: alertnow/README.md
Comment (buraizu) [inline alertnow/README.md:97]: These images don't seem to exist yet -- could you confirm if this is expected?

**Review by buraizu — CHANGES_REQUESTED** [review]
Thank you for following up, suggested some edits and also some questions about the images. Feel free to reach out with any questions.

File: alertnow/README.md
Comment (Uijong-Kim) [reply alertnow/README.md:106]: As shown in the URL below, you can see that the image is displayed normally in my repo.
If the image of the link included in the current README.md file is not displayed, it seems that the image has not been uploaded because the branch merge has not been done yet. Is there any additional action I need to execute?

https://raw.githubusercontent.com/Uijong-Kim/integrations-extras/add-alertnow-integration/alertnow/images/alertnow_overview.png

**Review by buraizu — CHANGES_REQUESTED** [review]
Thank you for incorporating these changes and sharing the images. I've made a couple of other small edits for clarity and concision, and also asked that you clarify the origin of the API key value in step 3 of the Datadog instructions. Thank you for your patience!

File: alertnow/README.md
Comment (Uijong-Kim) [reply alertnow/README.md:49]: That's right. I will change the "API-KEY" variable name so that it can be clearly recognized.

**Review by Uijong-Kim — COMMENTED** [review]
Changed the name of the placeholder variable.
before : API-KEY
after : ALERTNOW-API-KEY

**Review by Uijong-Kim — COMMENTED** [review]
Update Variables

File: alertnow/metadata.csv
Comment (yzhan289) [inline alertnow/metadata.csv:1]:
```suggestion
metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
```
We recently added this new column for `metadata.csv`.

**Review by Uijong-Kim — COMMENTED** [review]
Update metadata.csv file

**Comment by yzhan289** [comment]
@Uijong-Kim Thanks for updating the PR! Do you have a logo in PNG format we can use for the tile?

**Comment by Uijong-Kim** [comment]
> @Uijong-Kim Thanks for updating the PR! Do you have a logo in PNG format we can use for the tile?

What is the recommended image file size (width, height)? And how do I add or forward the file?

**Comment by yzhan289** [comment]
@Uijong-Kim We can take care of the actual file adding, but do you have a link to the logo? I think our teams will be able to resize if necessary.

**Review by Uijong-Kim — COMMENTED** [review]
AlertNow tile logo image

File: alertnow/images/IMAGES_README.md
Comment (sarah-witt) [inline alertnow/images/IMAGES_README.md:1]: Can you remove this file?

**Review by sarah-witt — COMMENTED** [review]
This looks great, I just had a small comment!

File: alertnow/manifest.json
Comment (sarah-witt) [inline alertnow/manifest.json:15]: One more thing, can you add a `CHANGELOG.md` file?

**Review by sarah-witt — CHANGES_REQUESTED** [review]

File: alertnow/manifest.json
Comment (sarah-witt) [inline alertnow/manifest.json:36]: Does this integration emit metrics? If so, can you add a metric to check (which will indicate whether the integration is installed or not)? If the integration does not send metrics, you can remove this section

File: alertnow/manifest.json
Comment (sarah-witt) [inline alertnow/manifest.json:5]: Do you mind moving this `classifier_tags` section under the `tile` section?

File: alertnow/manifest.json
Comment (sarah-witt) [inline alertnow/manifest.json:12]:
```suggestion
    "classifier_tags": [
        "Supported OS::Linux",
        "Supported OS::Mac OS",
        "Supported OS::Windows"
    ],
    "overview": "README.md#Overview",
```

File: alertnow/images/IMAGES_README.md
Comment (Uijong-Kim) [reply alertnow/images/IMAGES_README.md]: Yes, I deleted this file.

File: alertnow/manifest.json
Comment (Uijong-Kim) [reply alertnow/manifest.json:15]: Yes I added this file.

File: alertnow/manifest.json
Comment (Uijong-Kim) [reply alertnow/manifest.json:5]: OK, I will edit the file according to your comments.

File: alertnow/manifest.json
Comment (Uijong-Kim) [reply alertnow/manifest.json:41]: This integration does not collect or export metrics.

**Comment by steveny91** [comment]
@Uijong-Kim This looks fine to me. Before I merge this and finalize the release of the tile, is there a specific timeline we should wait for? Or can this be released already?

**Comment by Uijong-Kim** [comment]
@steveny91 I have applied all the reviews requested by reviewers and am waiting for a new review. I don't think there is any additional work I need to do at the moment.

**Pull request merged**
